include/ggl/attr.h

```diff
@@ -25,10 +25,8 @@
 #define VISIBILITY(v)
 #endif
 
-#ifdef GGL_SDK_EXPORT_API
-#define GGL_EXPORT VISIBILITY(default)
-#else
-#define GGL_EXPORT VISIBILITY(hidden)
+#ifndef GGL_EXPORT
+#define GGL_EXPORT
```

Comment:
Removing visibility tags generates worse code for the static library case, which is what we support and for which we optimize.

Reply:
We want every definition to be marked hidden if static and not exporting the API.